rebar.test.config

```diff
@@ -7,5 +7,5 @@
 {xref_checks, [undefined_function_calls]}.
 {deps_dir, "deps.test"}.
 {lib_dirs, ["deps"]}.
-{deps, [{riak_test, ".*", {git, "git://github.com/basho/riak_test", {branch, "master"}}}]}.
+{deps, [{riak_test, ".*", {git, "git://github.com/basho/riak_test", {branch, "riak/2.0"}}}]}.
```

Comment:
So, the master branch should work, but it's broken right now (checking out riak_test master and running get-deps fails). I switched the version to un-break the ensemble test target. Since the ensemble riak_test modules are stored in the ensemble repo, it doesn't really matter too much which version of riak_test we are using, as long as the version we use is compatible with the ensemble tests. 

I expect when we merge forward, we will merge the riak_test version change, which will fix the develop-2.2 tests. At some point when riak_test is fixed, maybe we'll want to switch ensemble back to using the riak_test master branch again.